android/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    compileSdkVersion 28
```

Comment:
Can you set this to use the style where it uses whatever the app tells it to?

Reply:
done